pom.xml

```diff
@@ -104,7 +108,7 @@
 
     <dependency>
       <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-junit5</artifactId>
+      <artifactId>quarkus-junit5-mockito</artifactId>
```

Comment:
Is this really used?

Reply:
It's not needed anymore as I changed the responses back to the given()... .expect() schema